napari/layers/_shapes_layer/shapes/ellipse.py

```diff
@@ -104,3 +105,44 @@ def to_mask(self, mask_shape=None):
         mask = poly_to_mask(mask_shape, self._face_vertices)
 
         return mask
+
+    def to_xml(self):
+        """Generates an xml element that defintes the shape according to the
+        svg specification.
+
+        Returns
+        ----------
+        element : Element
```

Comment:
3rd-party libraries should always specify the full (shortest) path to the object:
```suggestion
        element : xml.etree.ElementTree.Element
```

Reply:
fixed in all places